mhs/selenium_tests/data/templates/xml_IN150016UK05.mustache

```diff
@@ -21,7 +22,7 @@
         <author1 typeCode="AUT">
             <AgentSystemSDS classCode="AGNT">
                 <agentSystemSDS classCode="DEV" determinerCode="INSTANCE">
-                    <id root="1.2.826.0.1285.0.2.0.107" extension="{{asid}}"/>
+                    <id root="1.2.826.0.1285.0.2.0.107" extension="9{{asid}}"/>
```

Comment:
Why the '9'?
The asid is input from the configuration, and shouldn't need amending...

Reply:
@Nayira Any idea where this came from/if it's needed?